scss/mixins/_container.scss

```diff
@@ -1,9 +1,11 @@
 // Container mixins
 
 @mixin make-container($gutter: $container-padding-x) {
+  --#{$variable-prefix}gutter-x: #{$gutter};
```

Comment:
Double check if it isn't a regression

Reply:
In my opinion based on the tests I made, it isn't a regression. But as it initializes the variable, do we need to keep the second part of expression (#{$gutter}) line 7 and line 8 ?

Reply:
Got the same question, I would have done `var(--#{$variable-prefix}gutter-x);` instead of  `var(--#{$variable-prefix}gutter-x, #{$gutter});` but I kept the Bootstrap version just to avoid any regressions